src/item.cpp

```diff
@@ -1076,34 +1078,82 @@ std::string item::info(bool showtext, std::vector<iteminfo> &dump_ref) const
         it_tool* tool = dynamic_cast<it_tool*>(type);
 
         if ((tool->max_charges)!=0) {
-            std::string charges_line = _("Charges"); //;
+            int t_max;
+            const std::string t_ammo_name = _(ammo_name(tool->ammo_id).c_str());
+            std::string temp_fmt;
+            std::string charges_line = _("Charges");
             dump->push_back(iteminfo("TOOL",charges_line+ ": " + to_string(charges)));
```

Comment:
You could also use `string_format` directly here and get rid of the `charges_line` variable:
```C++
string_format( _("Charges: %d"), charges)
```